config/locales/en/settings.en.yml

```diff
@@ -0,0 +1,15 @@
+en:
+  settings:
+    settings: "Settings"
+  quickbooks:
+    accounts:
+      accounts: "QuickBooks Accounts"
+      note: "Note: These account settings will only apply to transactions created in the future."
+    connection:
+      connect_to_qb: "Connect to QuickBooks"
+      connected: "Connected"
+      connection: "QuickBooks Connection"
+      disconnect: "Disconnect"
+      disconnect_from_qb: "Disconnect from QuickBooks"
+      not_connected: "Not Connected"
+      status: "QuickBooks Status"
```

Comment:
When you refactor to move these fields into the form, some of this should move into the simple_form translations.